capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -464,11 +471,6 @@ public void onInterrupted() {
         });
     }
 
-    @VisibleForTesting
```

Comment:
[Clarification] why did we remove it? Don't we use it for tests?

Reply:
@zladzeyka I don't think we used this line of code anymore even in test cases:
` @VisibleForTesting
    PaymentQRCodeReader getPaymentQRCodeReader() {
        return mPaymentQRCodeReader;
    }`